lib/tessel/access-point.js

```diff
@@ -146,12 +151,16 @@ Tessel.prototype.createAccessPoint = function(opts) {
 
   return this.simpleExec(commands.getAccessPointSSID())
     .then(() => {
+      log.spinner.stop();
```

Comment:
Similar note as this comment -> https://github.com/tessel/t2-cli/pull/768/files#r66899373